lib/view_model/active_record/controller.rb

```diff
@@ -91,8 +96,34 @@ def prerender_viewmodel(...)
 
   private
 
+  # Viewmodels allow for integer or uuid ids
```

Comment:
I think it's been implicit up to now. The only reason I'm adding it here is because I couldn't straightforwardly make an `ArrayOf.new(Anything)`. That said, I think it would be just fine and sufficient to say "Array of Integer or String" and not have any further constraints, which would effectively be the same as before. Maybe that would be better.

Reply:
I loosened the restriction back to integer-or-string